yarn-project/end-to-end/src/e2e_key_registry.test.ts

```diff
@@ -143,7 +143,11 @@ describe('Key Registry', () => {
     it(`rotates npk_m with authwit`, async () => {
       const action = keyRegistry
         .withWallet(wallets[1])
-        .methods.rotate_npk_m(wallets[0].getAddress(), secondNewMasterNullifierPublicKey.toNoirStruct(), Fr.ZERO);
+        .methods.rotate_npk_m(
+          wallets[0].getAddress(),
+          secondNewMasterNullifierPublicKey.toWrappedNoirStruct(),
```

Comment:
It's a bit unfortunate that you need to know to use this specific converter and not the regular `toNoirStruct`. Perhaps also typing them in TS is a bit overblown, but something I'd definitely consider if we keep key rotation. Can you create an issue for this for us to tackle later if we do?

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/8124